src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -207,6 +239,32 @@ public int getInt()
         return INTEGER_CONVERTER.convert(value == null ? defaultVal : value);
     }
 
+    /**
+     * Gets the value of a system property as a int.
+     * @return system property int value if it exists, overrideDefaultValue otherwise.
+     */
+    public int getInt(int overrideDefaultValue)
+    {
+        String value = System.getProperty(key);
+        if (value == null)
+            return overrideDefaultValue;
+
+        return INTEGER_CONVERTER.convert(value);
+    }
+
+    public void setInt(int value)
+    {
+        System.setProperty(key, Integer.toString(value));
+    }
+
+    public Set<String> getSet()
```

Comment:
Not sure how often we will have to use this one but it looks nice to have probably

Reply:
atm it is a single case but I have seen other cases which are the same pattern, so as we add more properties should have a few more usages (though some are list I believe).